test/e2e/framework.go

```diff
@@ -1592,6 +1593,27 @@ func (data *TestData) runNetcatCommandFromTestPod(podName string, ns string, ser
 	return fmt.Errorf("nc stdout: <%v>, stderr: <%v>, err: <%v>", stdout, stderr, err)
 }
 
+func (data *TestData) runNetcatCommandFromTestPodWithProtocol(podName string, server string, port int32, protocol string) error {
```

Comment:
if you add this function, you should modify `runNetcatCommandFromTestPod` to invoke it, thus avoiding code duplication

Reply:
This is being used in `networkpolicy_test.go` as well, will it be fine if I make changes there too? It will involve adding an empty string as argument to all function calls, or I could also make the function accept protocol as `...string`.

Reply:
sorry I don't understand your comment.

I'm just suggesting this change to avoid code duplication:
```go
func (data *TestData) runNetcatCommandFromTestPod(podName string, ns string, server string, port int32) error {
    return data. runNetcatCommandFromTestPodWithProtocol(pdoName, ns, server, port, "tcp")
}
```